src/web/client/test/integration/portalSchemaReader.test.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import * as schemaHelperUtil from "../../utilities/schemaHelperUtil";
+import * as portalSchemaReadeMock from "../integration/portalSchemaReader.mock";
+import { schemaEntityKey } from "../../../client/schema/constants";
+import Sinon, { stub } from "sinon";
+import {
+    getEntitiesSchemaMap,
+    getDataSourcePropertiesMap,
+    getEntitiesFolderNameMap,
+} from "../../../client/schema/portalSchemaReader";
+import { expect, assert } from "chai";
+
+describe("portalSchemaReader", () => {
+    afterEach(() => {
+        Sinon.restore();
+    });
+    it("getEntitiesSchemaMap_withValidSchemaV2_shouldBeMap", () => {
+        //Action
```

Comment:
Don't comment action & assert. Its for internal understanding

Reply:
ok

Reply:
You want me to have Act , Action and Assert like this Assert , Act , Action . I am doing the same it's showing error.